tests/python/gpu/test_operator_gpu.py

```diff
@@ -292,15 +282,63 @@ def test_take_with_type():
                                         'take_a': 'write'},
                               arg_params=arg_params)
 
+def check_rnn_consistency(cell1, cell2):
+    dshape = (32, 5, 200)
+    data = mx.sym.Variable('data')
+
+    sym1, _ = cell1.unroll(5, data, merge_outputs=True)
+    mod1 = mx.mod.Module(sym1, label_names=None, context=mx.gpu(0))
+    mod1.bind(data_shapes=[('data', dshape)], label_shapes=None)
+
+    sym2, _ = cell2.unroll(5, data, merge_outputs=True)
+    mod2 = mx.mod.Module(sym2, label_names=None, context=mx.gpu(0))
+    mod2.bind(data_shapes=[('data', dshape)], label_shapes=None)
+
+    mod1.init_params()
+    args, auxs = mod1.get_params()
+    args = cell1.unpack_weights(args)
```

Comment:
@sbodenstein See here for example of training with fused and testing with unroll